frontend/app/routes/$lang+/_protected+/personal-information+/mailing-address+/edit.tsx

```diff
@@ -15,16 +16,16 @@ import { InputField } from '~/components/input-field';
 import type { InputOptionProps } from '~/components/input-option';
 import { InputSelect } from '~/components/input-select';
 import { getPersonalInformationRouteHelpers } from '~/route-helpers/personal-information-route-helpers.server';
-import { getAddressService } from '~/services/address-service.server';
+import { getAuditService } from '~/services/audit-service.server';
 import { getLookupService } from '~/services/lookup-service.server';
 import { getRaoidcService } from '~/services/raoidc-service.server';
```

Comment:
Can we add instrumentation (via `getInstrumentationService`) to this route (similar to the home address route) so we don't forget later on?

Reply:
added